samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -668,6 +668,7 @@ class SamzaContainer(
     try {
       info("Starting container.")
 
+      var startTime = System.nanoTime()
```

Comment:
`var` is used for mutable members. In this, you can simply use `val` because you are not going to modify `startTime`